modular_skyrat/modules/assault_operatives/code/armaments/armament_utility.dm

```diff
@@ -3,7 +3,7 @@
 	category_item_limit = 8
 
 /datum/armament_entry/assault_operatives/utility/suppressor
-	item_type = /obj/item/suppressor
+	item_type = /obj/item/suppressor/standard
```

Comment:
I feel like they can have the illegal one, considering they're from the Syndicate, but I'm not overly worried about it either.